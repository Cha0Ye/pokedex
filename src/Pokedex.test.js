import React from 'react';
import Pokedex from './Pokedex';
import toJson from "enzyme-to-json";
import { shallow, mount } from "enzyme";

it("renders using shallow", function() {
    shallow(<Pokedex />);
  });

it("renders using mount", function() {
    mount(<Pokedex />);
});

it("matches snapshot", function() {
    let wrapper = shallow(<Pokedex />);
    let serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
  });