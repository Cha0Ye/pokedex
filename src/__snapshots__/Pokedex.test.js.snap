// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <ul
    className="Pokedex"
  >
    <li>
      <Pokecard
        base_experience={62}
        id={4}
        name="Charmander"
        type="fire"
      />
    </li>
    <li>
      <Pokecard
        base_experience={63}
        id={7}
        name="Squirtle"
        type="water"
      />
    </li>
    <li>
      <Pokecard
        base_experience={72}
        id={11}
        name="Metapod"
        type="bug"
      />
    </li>
    <li>
      <Pokecard
        base_experience={178}
        id={12}
        name="Butterfree"
        type="flying"
      />
    </li>
    <li>
      <Pokecard
        base_experience={112}
        id={25}
        name="Pikachu"
        type="electric"
      />
    </li>
    <li>
      <Pokecard
        base_experience={95}
        id={39}
        name="Jigglypuff"
        type="normal"
      />
    </li>
    <li>
      <Pokecard
        base_experience={225}
        id={94}
        name="Gengar"
        type="poison"
      />
    </li>
    <li>
      <Pokecard
        base_experience={65}
        id={133}
        name="Eevee"
        type="normal"
      />
    </li>
  </ul>
</div>
`;
